### ISSUE TITLE
mermaid diagrams won't show up when a post loads blazingly fast

### ISSUE THREAD

**Issue by li6in9muyou** [cotes2020/jekyll-theme-chirpy#740]
## Checklist

- [x] I have read the [tutorials](https://cotes2020.github.io/chirpy-demo/categories/tutorial/) and know the correct effect of the functional design.
- [ ] There are no similar reports on [existing issues](https://github.com/cotes2020/jekyll-theme-chirpy/issues?q=is%3Aissue) (including closed ones).
- [x] I found the bug on the latest code of the `master` branch.

## Describe the bug

The bug is that mermaid diagrams won't show up. 

The cause is that mermaid is converting elements matching css selector `.mermaid` to diagrams however these elements are inserted at load event captured by jquery ([see mermaid.html#L47](https://github.com/cotes2020/jekyll-theme-chirpy/blob/e107a2e759836510216ea8567aa8b203decace31/_includes/mermaid.html#L47)). 

Sometimes, `.mermaid` elements are inserted before mermaid starting its initial render process and sometimes it's the other way around. As far as I know, there is no guaranteed way to reproduce this bug. 

The workaround presented [here](https://github.com/cotes2020/jekyll-theme-chirpy/issues/536#issuecomment-1159594179) does not work after step 4 (see below).
`mermaid.initialize()` call does not seem to kicks off the rendering process which contradicts a comment made by repo owner in issue #536 ([this comment](https://github.com/cotes2020/jekyll-theme-chirpy/issues/536#issuecomment-1074331797)). 

Adding a `mermaid.init()` call can fix this bug, see [this commit](https://github.com/li6in9muyou/issue-not-triggering-mermiad-render/commit/980be35efeabd58a368bb7ee9c7f482b902976df).

### To Reproduce

1. clone master branch of this repo
2. add a new post with nothing but a simple mermaid diagram.
3. comment out webfonts in `/_data/assets/cross_origin.yml`
4. set theme to dark in `/_config.yml`
5. build site and serve
6. go to that newly added post

For example [this commit](https://github.com/li6in9muyou/issue-not-triggering-mermiad-render/commit/c29110f1) has completed steps 1,2,3,4.

### Expected behavior

Expected behavior is that one diagram rendered by mermaid can be seen. Actual behavior is that one line of mermaid syntax is shown.

### Environment
I use jekyll official image to build this site locally i.e. `docker pull jekyll/jekyll`.

| Command                           | Version |
|-----------------------------------|---------|
| `ruby -v`                         |         |
| `gem -v`                          |         |
| `bundle -v`                       |         |
| `bundle exec jekyll -v`           |         |
| `bundle info jekyll-theme-chirpy` |         |

**Comment by cotes2020** [comment]
Duplicate of #536
